README.md

```diff
@@ -502,7 +503,16 @@ $ remme batch get-list \
         }
     ]
 }
-
+$ remme batch get-list \
```

Comment:
At this section where we describe only `--ids-only`. So we should remove excess parameters. I mean refactor to:

```
$ remme batch get-list --ids-only --node-url=node-6-testnet.remme.io
{
    "result": [
        "e4d5089f2ef1...6b0f185b3b6ca",
        "df5e555f.317f039501",
        ...
    ]
}
```

Reply:
Done.